annif/annif.py

```diff
@@ -32,10 +33,15 @@
         }
 
 
-def parseIndexname(projectid):
+def parse_indexname(projectid):
```

Comment:
shouldn't this be `format_indexname` or even better `format_index_name`?

Reply:
Yep, fixing.